### DISCUSSION TITLE
Building from sources and not using qt creator

### DISCUSSION THREAD

**Discussion by eanisutkin** [QtExcel/QXlsx#189] [Q&A]
Hi, I am currently evaluating the library on the point of using it in an environment with no Qt creator at all.

However I've run into several problems: 

1) Usage demo implies QtCreator, which i don't have at all, I do however have qt build as shared libraries strait from sources. Currently it looks like i won't be able to use this library

2) This is a statically linked code, however i can't fandom why. Static lib is not always good, for example I needed a shared variant and had to figure out how to rebuilt it. A guide for building a shared library with changeble target path would be welcome

**Comment by j2doll** [comment]
Dear @eanisutkin 

Sorry for late reply.

The reason why I used the library method was because of the version control for dll(so) after distribution. 

So I also proceeded with the method(pri) that does not use lib(a) and dll(so) at all. 

I will also consider what you said. 

Thank you for your good opinion. 😃
